src/sass/pages/index.scss

```diff
@@ -0,0 +1,44 @@
+.home-hero {
+  @include corporate-gradient;
+  color: $color-hero-text;
+  padding: $padding-hero-vertical 0;
+
+  &__wrap {
+    @include row;
+    @include container;
+  }
+
+  &__info {
+    @include columns(5);
+
+    h1 {
+      font-size: $font-size-title-big;
+      text-transform: lowercase;
+      line-height: 1.2em;
+    }
+
+    p {
+      margin: $margin-hero-desc-vertical 0;
+      line-height: 1.3em;
+    }
+
+    &__links {
+      a {
+        text-transform: lowercase;
```

Comment:
I'm waiting on the response of the designer to know if all of them are going to be lowercase. If they are, yes, we can extract this to something else.